docs/appendices/environment_vars.rst

```diff
@@ -139,7 +139,7 @@ There are several environment variables that affect the way Toil runs.
 | TOIL_SLURM_ARGS                  | Arguments for sbatch for the slurm batch system.   |
 |                                  | Do not pass CPU or memory specifications here.     |
 |                                  | Instead, define resource requirements for the job. |
-|                                  | There is no default value for this variable.       |
+|                                  | By default, "--export=ALL" is set.                 |
```

Comment:
This makes it sound like if `TOIL_SLURM_ARGS` is not set, it has the same effect as setting it to `--export=ALL`. In fact, `--export=all` is supposed to be set whether `TOIL_SLURM_ARGS` is set or not. So you should elaborate here and explain how if the `--export` or `--export-file` options are used here, they will override the default behavior of exporting all environment variables.